packages/manager/modules/pci/src/projects/project/data-processing/jobs/data-processing-jobs.html

```diff
@@ -0,0 +1,124 @@
+<div data-ui-view>
+    <div>
+        <!-- cui messages -->
+        <cui-message-container
+            data-messages="$ctrl.messages"
+        ></cui-message-container>
+        <oui-datagrid
+            data-rows-loader="$ctrl.getJobs($config)"
+            data-refreshable
+            criteria="$ctrl.criteria"
+            on-criteria-change="$ctrl.onListParamChange($criteria)"
+        >
+            <oui-datagrid-column
+                data-title="::'data_processing_job_name' | translate"
+                data-property="name"
+                data-sortable
+                data-type="string"
+                data-searchable
+                data-filterable
+            >
+                <oui-button
+                    data-variant="link"
+                    data-on-click="$ctrl.showJob($row.id)"
+                >
+                    <span data-ng-bind="::$row.name"></span>
+                </oui-button>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_job_id' | translate"
+                data-property="id"
+                data-sortable
+                data-type="string"
+                data-filterable
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_region' | translate"
+                data-property="region"
+                data-type="string"
+                data-sortable
+                data-searchable
+                data-filterable
+            >
+                <span
+                    data-ng-bind="::$ctrl.ovhManagerRegionService.getTranslatedMicroRegion($row.region)"
+                ></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_job_type' | translate"
+                data-property="type"
+                data-sortable
+                data-searchable
+                data-filterable
+            >
+                <span data-ng-bind="::$row.type"></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title="::'data_processing_job_cores_ram' | translate"
+                data-property="vcores"
+                data-sortable
+            >
+                <span
+                    >{{ $row.vcores }}{{ ::
+                    'data_processing_job_cores_abbreviation' | translate }}/{{
+                    $row.ram }}</span
+                >
```

Comment:
`<span data-ng-bind="$row.vcores"></span>
<span data-translate="data_processing_job_cores_abbreviation"></span>
<span>/<span>
<span data-ng-bind="$row.ram"></span>`

Reply:
Unfortunately, doing this translation like this add spacing between all the span which is something that we do not want. Would it be ok to use a data-translate with a data-translate-values ?